build/e2e-image/entrypoint.sh

```diff
@@ -25,13 +27,13 @@ if [ "$1" = 'local' ]
 then
         gcloud auth login
 fi
-gcloud container clusters get-credentials e2e-test-cluster \
+gcloud container clusters get-credentials $TEST_CLUSTER_NAME \
         --zone=us-west1-c --project=agones-images
 kubectl port-forward statefulset/consul 8500:8500 &
 echo "Waiting consul port-forward to launch on 8500..."
 timeout 60 bash -c 'until printf "" 2>>/dev/null >>/dev/tcp/$0/$1; do sleep 1; done' 127.0.0.1 8500
 echo "consul port-forward launched. Starting e2e tests..."
-echo "consul lock -child-exit-code=true -timeout 30m -try 30m -verbose LockE2E '/root/e2e.sh "$FEATURES""
+echo "consul lock -child-exit-code=true -timeout 30m -try 30m -verbose LockE2E '/root/e2e.sh "$FEATURES"'"
```

Comment:
Is this apostrophe correct? Should there be a partner somewhere?

Reply:
It should result in `'/root/e2e.sh $FEATURES'`

Reply:
Note that this is `echo` command. From Cloud build:
```
Step #22 - "e2e-stable": Handling connection for 8500
Step #22 - "e2e-stable": consul port-forward launched. Starting e2e tests...
Step #22 - "e2e-stable": consul lock -child-exit-code=true -timeout 30m -try 30m -verbose LockE2E '/root/e2e.sh '
Step #21 - "e2e-feature-gates": Handling connection for 8500
Step #21 - "e2e-feature-gates": consul port-forward launched. Starting e2e tests...
Step #21 - "e2e-feature-gates": consul lock -child-exit-code=true -timeout 30m -try 30m -verbose LockE2E '/root/e2e.sh PlayerTracking=true&ContainerPortAllocation=true'
```